hat/assets/js/apps/Iaso/domains/home/hooks/useHomeButtons.tsx

```diff
@@ -66,6 +67,12 @@ export const useHomeButtons = (): Button[] => {
                     Icon: <Assignment />,
                     url: `/${baseUrls.planning}`,
                 },
+                {
+                    label: formatMessage(MESSAGES.admin),
+                    permissions: paths.projectsPath.permissions,
+                    Icon: <PhonelinkSetupIcon />,
+                    url: `/${baseUrls.projects}`,
```

Comment:
Is this correct ? this should link to projects ?

Reply:
In the description of [the ticket](https://bluesquare.atlassian.net/browse/IA-3306), it mentioned **the icon can be a smartphone**. 

So, I suppose, the redirect URL should be the same as Projects(same icon) in the menu on the the following screenshot:

![Iaso-Projects](https://github.com/user-attachments/assets/957b4f2b-abdc-4cee-b62f-0e4a3d19a9ff)